business_requirement_activity_report/report/timesheet_activity_report.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#   2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.osv import fields, osv
+from openerp import tools
+
+
+class TimesheetReport(osv.Model):

```

Comment:
@xie8899 is totally necessary to use old API or could be done on new API so the possible porting it will be easier?